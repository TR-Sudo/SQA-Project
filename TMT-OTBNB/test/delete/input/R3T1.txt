Login
AdminUser
Delete
AdminUser
username05
Logout
quit