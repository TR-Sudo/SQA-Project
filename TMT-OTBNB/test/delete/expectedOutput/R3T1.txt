Enter transaction: Enter username: You have been logged in
Enter transaction: Enter username: You cannot delete your own account
Enter username: Deleted user username05
Enter transaction: You have been logged out
Enter transaction: 